python/docs/source/_static/css/pyspark.css

```diff
@@ -24,7 +24,7 @@ body {
 }
 
 h1,h2 {
-    color:#1B5162!important;
+    color:#17A2B8!important;
```

Comment:
This hex value references a [value from Pandas](https://github.com/pandas-dev/pandas/blob/main/doc/source/_static/css/pandas.css#L5) who uses the same theme as ours:

```css
:root {
  /* Use softer blue from bootstrap's default info color */
  --pst-color-info: 23, 162, 184;
}
```

The hex value corresponding to RGB values ​​`23, 162, 184` is `#17A2B8`.